### PR TITLE
Feature/club table

### PR DESCRIPTION
**Pull request by mikefgalvin**
## Description: 
- What work was done?
Added a club model, controller, service, and service implementation
Added a list all clubs endpoint
Added a member model, controller, service, and service implementation
Added a list all members endpoint

- Why was this work done?
Building out the initial structure of our backend database
Letting the frontend pull a list of information from either table
 
- What feature / user story is it for?
As a Super Admin, Club Director, or YDP, I would like to store and have access to member and club information

- Any relevant links or images / screenshots

- User story link (Trello card)
https://trello.com/c/N7PvlpNR

## Type of change

- [X] New feature
- [ ] Bug fix
- [ ] Design change
- [ ] Modified existing code

## Minimum reviewers
- [x] 2 reviewers

## Testing
- [ ] Testing completed

### CODE REVIEW

File: src/main/java/com/lambdaschool/oktafoundation/OktaFoundationApplication.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
  */
 @EnableJpaAuditing
 @SpringBootApplication
-@PropertySource(value = "file:/Users/lambdajohn/oktajavaconf.properties", ignoreResourceNotFound = true)
+@PropertySource(value = "file:/Users/Galvin/oktafoundationconf.properties", ignoreResourceNotFound = true)
```
Comment (ginabethrussell) [inline src/main/java/com/lambdaschool/oktafoundation/OktaFoundationApplication.java]: Remove local conf.properties file from main branch.

File: src/main/java/com/lambdaschool/oktafoundation/controllers/ClubController.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+@RequestMapping("/clubs")
+public class ClubController
+{
+    @Autowired
+    ClubRepository clubRepository;
+
+    @Autowired
+    private ClubService clubService;
+
+    @PreAuthorize("hasAnyRole('ADMIN')") // Double Check ROLE
```
Comment (ginabethrussell) [inline src/main/java/com/lambdaschool/oktafoundation/controllers/ClubController.java:26]: Change to SUPERADMIN and CLUBDIR

File: src/main/java/com/lambdaschool/oktafoundation/controllers/MemberController.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+@RequestMapping("/members")
+public class MemberController
+{
+    @Autowired
+    MemberRepository memberRepository;
+
+    @Autowired
+    MemberService memberService;
+
+    @PreAuthorize("hasAnyRole('ADMIN')")
```
Comment (ginabethrussell) [inline src/main/java/com/lambdaschool/oktafoundation/controllers/MemberController.java:31]: Change to SUPERADMIN and CLUBDIR

File: src/main/java/com/lambdaschool/oktafoundation/models/Club.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+public class Club extends Auditable
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @NotNull
+    @Column(unique = true)
+    private String clubname;
+
```
Comment (ginabethrussell) [inline src/main/java/com/lambdaschool/oktafoundation/models/Club.java:21]: Remove location field

File: src/main/java/com/lambdaschool/oktafoundation/models/Club.java [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    }
+
+    public String getClubname() {
+        return clubname;
+    }
+
+    public void setClubname(String clubname) {
+        this.clubname = clubname;
+    }
+
```
Comment (ginabethrussell) [inline src/main/java/com/lambdaschool/oktafoundation/models/Club.java:50]: Remove location getters and setters

**Review by ginabethrussell — APPROVED** [review]
Great work adding member and club models, services, repos, and controllers. Make changes to roles, and remove location from club model.

**Review by justin-mavity — COMMENTED** [review]
Looks good, good job!

**Pull request merged**